package.json

```diff
@@ -26,7 +26,9 @@
     "test:ui": "babel-node test/main.js",
     "bundlesize": "bundlesize",
     "lokalise:download": "lokalise2 --token $LOKALISE_TOKEN --project-id $LOKALISE_PROJECT_ID file download --format json --unzip-to ./src/locales/ && node ./scripts/unwrap_lokalise.js",
-    "license-checker": "license-checker-rseidelsohn --json > licenses.json --relativeLicensePath --relativeModulePath && prettier --write ./licenses.json"
+    "license-checker": "license-checker-rseidelsohn --json > licenses.json --relativeLicensePath --relativeModulePath && prettier --write ./licenses.json",
+    "mock-server:build": "cd ./test/mock-server && ./prepare.sh && docker build -t onfido-web-sdk:ui-mock-server .",
+    "mock-server:run": "docker run -it -d --init --rm -p 8080:8080 onfido-web-sdk:ui-mock-server"
```

Comment:
We should probably update the `CONTRIBUTING.md` file as docker is required to run these commands

Reply:
Added the prerequisites section in CONTRIBUTING.md